resource-aggregate/cqrs/eventstore/test/events.go

```diff
@@ -13,8 +13,7 @@ func MakeResourceLinksPublishedEvent(resources []*commands.Resource, deviceID st
 		Resources: resources,
 		DeviceId:  deviceID,
 		AuditContext: &commands.AuditContext{
```

Comment:
Audit context doesn't exist anymore, or?

Reply:
It exist - only deviceID was remove there. Authourization context was dropped.